recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-aos/aos-iamanager/aos-iamanager_git.bbappend

```diff
@@ -28,6 +28,11 @@ FILES_${PN} += " \
     /var/aos/finish.sh \
 "
 
+do_compile_prepend(){
```

Comment:
Is it still required after moving to go 1.14?

Reply:
Requires dunfell release on domd yocto